### ISSUE TITLE
Error running Morpheus v0.0.5 on macOS Sonoma v14.2 (8gb RAM)

### ISSUE THREAD

**Issue by Dannyz123** [MorpheusAIs/Morpheus#169]
When running without Ollama, OS crashes entirely. Tried installing Ollama and running it with Orca-mini. This is the error that pops-up. Is there a way to fix this?

<img width="1080" alt="Screenshot 2023-12-13 at 12 00 37 PM" src="https://github.com/MorpheusAIs/Morpheus/assets/153759228/c47b544d-ce34-4da0-9d0a-dbccfcccdd44">

**Comment by DavidAJohnston** [comment]
Yeah the default Mistral model is 14 gigs so 8 gigs would give you an error. It would be good to have Morpheus start with a smaller model like Orca-Mini and upgrade from there based on user hardware.

**Comment by Dannyz123** [comment]
I figured that was the issue. Still thought it was worth mentioning.

**Comment by CryptoPol** [comment]
I have the same issue. Could somebody help me to have Morpheus start with Orca-Mini ?
